pkg/drivers/kic/types.go

```diff
@@ -23,11 +23,13 @@ import (
 )
 
 const (
-	// Version is the current version of kic
+	// Version is the current version of kic -- also used as image tag
 	Version = "v0.0.36-1673540226-15630"
 
-	// SHA of the kic base image
-	baseImageSHA = "03c9592728381094cbd0ff9603f75ae6b485dd7a390c3e35f02ae5ec10f2f3ad"
+	// SHA of compressed image, used as a registry reference -- more reliable than a tag
```

Comment:
For the minikube images, there is nothing that makes the digests more reliable than the tags.

Reply:
I meant it in a general way: a tag can theoretically point to another (newer?)image, possibly breaking workflows.
minikube could not have this kind of problem.